WordPress/gutenberg
# Hide the theme without comments.php deprecation message

closes #28045 

> Block-based Themes throw up a depreciation warning if they do not include a `comments.php` file:
> 
> > Deprecated: Theme without comments.php is deprecated since version 3.0.0 with no alternative available. Please include a comments.php template in your theme. in /var/www/html/wp-includes/functions.php on line 5059

I'm not very proud of what I did to achieve the removal of this deprecation but at the moment, I don't see any other way to do it. Open to suggestions

**Testing instructions**

 - Have an FSE theme without comments.php file
 - Add a "Post comments" block to any post
 - Check the frontend
 - You shouldn't see the deprecation message.
